### PR TITLE
Xenos can now select a tunnel from the minimap instead of a list

### PR DESCRIPTION
**Pull request by ivanmixo**
## About The Pull Request

https://github.com/tgstation/TerraGov-Marine-Corps/assets/22431091/f135ffc9-60a7-41c8-b93b-6e708aaf7335
## Why It's Good For The Game

Being able to just click where you want to go to instead of trying to decipher what the hivelord meant by "asdfsd" is better.
## Changelog
:cl:
qol: You can now click the tunnel you want to go to on the minimap instead of picking from a list
/:cl:

### CODE REVIEW

**Comment by Lumipharon** [comment]
Unreasonably based.

File: code/modules/xenomorph/xeno_structures.dm [hunk 1]
```diff
@@ -397,26 +397,41 @@ TUNNEL
 
 ///Here we pick a tunnel to go to, then travel to that tunnel and peep out, confirming whether or not we want to emerge or go to another tunnel.
 /obj/structure/xeno/tunnel/proc/pick_a_tunnel(mob/living/carbon/xenomorph/M)
-	var/obj/structure/xeno/tunnel/targettunnel = tgui_input_list(M, "Choose a tunnel to crawl to", "Tunnel", GLOB.xeno_tunnels_by_hive[hivenumber])
+	to_chat(M, span_notice("Select a tunnel to go to."))
```
Comment (QualityVan) [inline code/modules/xenomorph/xeno_structures.dm:400]: Is there a way to cancel out once you've opened the map, or is it just click somewhere with no tunnels to not go anywhere?

Comment (ivanmixo) [reply code/modules/xenomorph/xeno_structures.dm:400]: no

Comment (ivanmixo) [reply code/modules/xenomorph/xeno_structures.dm:400]: You can click the edge of the map to exit tho yea

**Comment by timothymtorres** [comment]
Amazing! This is long overdue.

**Comment by AusarAtkin** [comment]
Rare moment where ivan is not cringe, do you need to click on the exact T sign?

**Comment by ivanmixo** [comment]
Within 10 tiles of the tunnel you want to go to

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**